src/pages/SigninPage/index.tsx

```diff
@@ -2,52 +2,87 @@ import React, { FC } from 'react';
 import { useHistory } from 'react-router-dom';
 
 import { authApi } from '@/apis';
+import { useSetUserState } from '@/atoms/userState';
 import Jumbotron from '@/components/common/Jumbotron';
 import Logo from '@/components/common/Logo';
-import Shell from '@/components/shell/Shell';
+import ShellCommand from '@/components/shell/ShellCommand';
+import Shell from '@/components/shell/ShellCopy';
+import { reject, renderProps, resolve } from '@/components/shell/utils';
```

Comment:
이게 만약 라이브러리로 제공되는 그러한 컴포넌트라면 이러한 형식으로 util이라는 이름이 좀 애매하지 않을까요 ?!

Reply:
으음... 그렇다면 helper 는 어떤가요!?

```suggestion
import { reject, renderProps, resolve } from '@/components/shell/helper';
```